[BUG]  GitHub Action에서 ./gradlew build 명령 실행 오류

**🐞 버그 설명**
GitHub Action에서 ./gradlew build 명령시 오류가 발생하는 문제가 있었습니다.

**📋 재현 방법**
```bash
Run ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-8.8-bin.zip
.............10%.............20%.............30%.............40%.............50%.............60%..............[7](https://github.com/Leets-Official/commitato-BE/actions/runs/9973554968/job/27559138986#step:5:8)0%.............[8](https://github.com/Leets-Official/commitato-BE/actions/runs/9973554968/job/27559138986#step:5:9)0%.............90%.............100%

Welcome to Gradle 8.8!

Here are the highlights of this release:
 - Running Gradle on Java 22
 - Configurable Gradle daemon JVM
 - Improved IDE performance for large projects

For more details see https://docs.gradle.org/8.8/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Task :compileJava
Note: /home/runner/work/commitato-BE/commitato-BE/src/main/java/com/leets/commitatobe/domain/login/usecase/LoginCommandServiceImpl.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :processResources
> Task :classes
> Task :resolveMainClassName
> Task :bootJar
> Task :jar
> Task :assemble
> Task :compileTestJava
> Task :processTestResources NO-SOURCE
> Task :testClasses


1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
> Task :test FAILED

CommitatoBeApplicationTests > contextLoads() FAILED
    java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java:180
        Caused by: org.springframework.boot.context.config.ConfigDataResourceNotFoundException at ConfigDataResourceNotFoundException.java:[9](https://github.com/Leets-Official/commitato-BE/actions/runs/9973554968/job/27559138986#step:5:10)7
            Caused by: software.amazon.awssdk.core.exception.SdkClientException at SdkClientException.java:[11](https://github.com/Leets-Official/commitato-BE/actions/runs/9973554968/job/27559138986#step:5:12)1

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.8/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
7 actionable tasks: 7 executed
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/commitato-BE/commitato-BE/build/reports/tests/test/index.html

* Try:
> Run with --scan to get full insights.

BUILD FAILED in 38s
Error: Process completed with exit code 1.
```

**⚙️ 기대 동작**
GitHub Acrion에서 gradle 빌드가 성공해야합니다.

**🖼️ 스크린샷**
코드로 대체 합니다.

**💻 데스크탑 (다음 정보를 기입해주세요):**
GitHub Action에서 발생하는 문제이기 때문에 생략합니다.

**📱 스마트폰 (다음 정보를 기입해주세요):**
없습니다.

**📜 추가 내용**
추가적으로 빌드가 되지 않으면 알려주시기 바랍니다.

--- a/build.gradle
+++ b/build.gradle
@@ -51,10 +51,6 @@ dependencies {
     // Database 연결
     implementation 'com.h2database:h2:2.1.210' // 개발 및 테스트를 위한 H2 인메모리 데이터베이스
 
-    // AWS
-    implementation platform("io.awspring.cloud:spring-cloud-aws-dependencies:3.1.0")
-    implementation 'io.awspring.cloud:spring-cloud-aws-starter-parameter-store'
-
     // GSON
     implementation "com.google.code.gson:gson:2.8.9"
 }

--- a/src/main/resources/application-dev.yaml
+++ b/src/main/resources/application-dev.yaml
@@ -2,11 +2,10 @@ spring:
   config:
     activate:
       on-profile: dev
-    import: aws-parameterstore:/Dev/
   datasource:
-    url: ${DBServer/MySQL/DB_PATH}
-    username: ${DBServer/MySQL/DB_USER}
-    password: ${DBServer/MySQL/DB_PASSWORD}
+    url: ${DB_PATH}
+    username: ${DB_USER}
+    password: ${DB_PASSWORD}
     driver-class-name: com.mysql.cj.jdbc.Driver
   jpa:
     show-sql: true
@@ -27,5 +26,5 @@ spring:
             scope: read:user
             redirect-uri: "http://localhost:8080/login/callback"
 
-  jwt:
-    secret: ${JWT/JWT_SECRET}
+jwt:
+  secret: ${JWT_SECRET}

